docs/en/latest/discovery/nacos.md

```diff
@@ -29,8 +29,47 @@ The performance of this module needs to be improved:
 
 ### Configuration for Nacos
 
+There are two ways to configure Nacos. Single instance and Multi instance method.
+
 Add following configuration in `conf/config.yaml` :
 
+#### Multi Instance method
```

Comment:
Put the single instance before multi, so the reading flows better because 

<img width="676" alt="image" src="https://github.com/user-attachments/assets/abb9d6a4-5b4c-459b-a9a2-0bc3bafaef7d" />